cabal.project

```diff
@@ -49,6 +52,7 @@ allow-newer:
            -- The following two dependencies are needed by plutus.
            , eventful-sql-common:persistent
            , eventful-sql-common:persistent-template
+           , formatting
```

Comment:
Can we scope this to a particular package like the other examples in this stanza? Also add an explanatory comment.

Reply:
Can `allow-newer` be scoped to a particular package somehow? It seems like you can only define it in `cabal.project`.

Reply:
`eventful-sql-common:persistent` relaxes the constraint on `persistent` _from_ `eventful-sql-common`, and nowhere else. So you only discard the bounds that you really need to, which is a bit safer. So the question is: which package has the overly tight bound on `formatting`? then we should put `X:formatting` here.

Reply:
I tried to limit it to `marlowe-symbolic` but for some reason it is not working. Could it be because there is no bound specified on `formatting` in `marlowe-symbolic.cabal` (the version bound is `-any`) ?

Reply:
Well then, that's not the problem. What solver error do you get without this?

Reply:
It uses the older version of the library and I get a compilation error (like I did when I had no allow-newer for formatting).

Reply:
Try putting instead a constraint that requires the newer version and then cabal will hopefully tell you what's requiring the old version and then you can relax *that*.